firmware/controllers/algo/launch_control.cpp

```diff
@@ -198,20 +195,26 @@ void setDefaultLaunchParameters(DECLARE_CONFIG_PARAMETER_SIGNATURE) {
 	engineConfiguration->enableLaunchBoost = true;
 	engineConfiguration->launchSmoothRetard = true; //interpolates the advance linear from launchrpm to fully retarded at launchtimingrpmrange
 	engineConfiguration->antiLagRpmTreshold = 3000;
+
 }
 
 void applyLaunchControlLimiting(bool *limitedSpark, bool *limitedFuel DECLARE_ENGINE_PARAMETER_SUFFIX) {
-
-	if (retardThresholdRpm < GET_RPM()) {
+	if (( engine->isLaunchCondition ) && ( retardThresholdRpm < GET_RPM() )) {
 		*limitedSpark = engineConfiguration->launchSparkCutEnable;
 		*limitedFuel = engineConfiguration->launchFuelCutEnable;
+	} else {
```

Comment:
I don't think you want this else case.  This implementation will break the normal rev limiter.

Reply:
Look at how it's called:
<img width="896" alt="image" src="https://user-images.githubusercontent.com/568254/100540139-3cfc1e80-31f0-11eb-8ab6-99654d659e4a.png">

Reply:
You are right, i've got that because of unit_test, but i have to modify